client/src/app/_helpers/error.handler.ts

```diff
@@ -2,42 +2,51 @@ import { ERRORS400 } from './errors.400';
 import { Router } from '@angular/router';
 import { ToasterService } from './../_services';
 import { HttpErrorResponse } from '@angular/common/http';
-import { Injectable, ErrorHandler, Injector } from '@angular/core';
+import { ErrorHandler, Injectable, Injector } from '@angular/core';
 
 @Injectable()
 export class ErrorsHandler implements ErrorHandler {
-
     constructor(
         // Because the ErrorHandler is created before the providers,
         // we’ll have to use the Injector to get them.
-        private injector: Injector
-    ) { }
+        private injector: Injector) {}
 
-    handleError(error: Error | HttpErrorResponse) {
+    handleError(error: Error | HttpErrorResponse): void {
         const toasterService = this.injector.get(ToasterService);
         const router = this.injector.get(Router);
 
-        if (error instanceof HttpErrorResponse) {
-            // Server or connection error happened
-            if (!navigator.onLine) {
-                // Handle offline error
+        if (error instanceof HttpErrorResponse)
+            if (!navigator.onLine)
+                // Server or connection error happened
                 return toasterService.showToaster('Pas de connexion internet');
-            } else {
+            else
                 switch (error.status) {
                     case 400:
-                        return toasterService.showToaster(ERRORS400[error.error.error]);
+                        return toasterService.showToaster(
+                            ERRORS400[error.error.error]
+                        );
                     case 401:
                         router.navigate(['/login']);
-                        return toasterService.showToaster('Opération non autorisée, redirection ...');
+
+                        return toasterService.showToaster(
+                            'Opération non autorisée, redirection ...'
+                        );
                     case 403:
-                        return toasterService.showToaster('Autorisations non suffisante pour effectuer la requête');
+                        return toasterService.showToaster(
+                            'Autorisations non suffisante pour effectuer la requête'
+                        );
                     case 404:
-                        return toasterService.showToaster('Appel serveur inconnu');
+                        return toasterService.showToaster(
+                            'Appel serveur inconnu'
+                        );
                     case 500:
-                        return toasterService.showToaster('Erreur serveur (indépendant du client). Merci de vérifier le serveur.');
+                        return toasterService.showToaster(
+                            'Erreur serveur (indépendant du client). Merci de vérifier le serveur.'
+                        );
+                    default:
+                        return;
                 }
-            }
-        } else {
+        else {
```

Comment:
So the if and the else if do not have braces, but the last one have?
I think it's really more error prone than before